python/tests/test_bindings.py

```
@@ -0,0 +1,93 @@
+
+def _make_test_data(cfg):
+    input_ids = list(range(cfg.seq_len))
+    target_ids = [(t + 1) % cfg.vocab_size for t in range(cfg.seq_len)]
+    return input_ids, target_ids
+
+
+def test_forward_returns_loss():
+    cfg = nl_hecate.SWAConfig(64, 4, 16, 24, 16, 256)
+    params = nl_hecate.init_params(cfg, 42)
+    input_ids, target_ids = _make_test_data(cfg)
+    loss, cache = nl_hecate.forward(params, cfg, input_ids, target_ids)
```

Ruff (0.15.0)

[warning] Unpacked variable `cache` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)